src/status/forcevote.py

```diff
@@ -16,30 +19,30 @@
 # FORCED_TARGETS is used to see which targets are being force-voted for.
 # If multiple calls to force_vote() force different targets, the union of all of those targets is taken.
 
-FORCED_COUNTS = UserDict() # type: UserDict[users.User, int]
-FORCED_TARGETS = UserDict() # type: UserDict[users.User, UserSet]
+FORCED_COUNTS: UserDict[User, int] = UserDict()
+FORCED_TARGETS: UserDict[User, UserSet] = UserDict()
 
-def _add_count(var, votee : users.User, amount : users.User) -> None:
+def _add_count(var, votee : User, amount : int) -> None:
```

Comment:
bad annotation format, no space before colons (yes I know it was that way before, but you can fix it since you're already touching those lines)